pkg/docker/docker.go

```diff
@@ -171,7 +188,7 @@ func (rco RunContainerOptions) asDockerCreateContainerOptions() docker.CreateCon
 	config := rco.asDockerConfig()
 	hostConfig := rco.asDockerHostConfig()
 	return docker.CreateContainerOptions{
-		Name:       "",
+		Name:       containerName("", ""),
```

Comment:
We cannot have name collisions. Keep in mind there can be several concurrent builds.